source/frontend/src/features/leases/hooks/useLeaseDetail.ts

```diff
@@ -129,7 +116,7 @@ export function useLeaseDetail(leaseId?: number) {
     lease,
     setLease,
     refresh: async () => {
-      setStaleLastUpdatedBy(true);
+      // setStaleLastUpdatedBy(true);
```

Comment:
Is there a reason for this? this looks like it will break the last update by logic.

Reply:
this was causing a useEffect infinite loop. the 'setStaleLastUpdatedBy(true)' is triggered by the children components.

Reply:
I would just remove this then, commenting out code shouldn't be necessary given that we have git history.

Reply:
removed